lib/GeometryMath.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+var Cesium = require('cesium');
+var Cartesian3 = Cesium.Cartesian3;
+var defined = Cesium.defined;
+var DeveloperError = Cesium.DeveloperError;
+
+module.exports = {
+    sumBarycentric: sumBarycentric,
+    flattenTriangle: flattenTriangle,
+    pointLineDistanceParametric: pointLineDistanceParametric
+};
+
+// Functions for working with geometry like triangles and lines
+
+var sumBarycentricScratch = new Cartesian3();
+function sumBarycentric(barycentric, vector0, vector1, vector2, result) {
+    //>>includeStart('debug', pragmas.debug);
+    if (!defined(barycentric)) {
+        throw new DeveloperError('in GeometryMath: barycentric is required');
```

Comment:
Get rid of `in GeometryMath: ` throughout your error messages, that's what callstacks are for.